src/bp/ui-shared/src/Toaster/index.tsx

```diff
@@ -49,12 +50,20 @@ const warningOptions = {
   key: 'warning'
 }
 
-const dismiss = (key: string) => {
+const selectToaster = (options?: IToasterProps) => {
+  if(options){
+    return Toaster.create(options)
+  } else{
+    return defaultToaster
+  }
+}
```

Comment:
Could be a one-liner: `const getToaster = (options?) => options ? Toaster.create(options) : defaultToaster`